VitaminJProcessing.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun  4 02:05:08 2024
+
+@author: Anupama Rajendra
+"""
+import csv
+import numpy as np
+import pandas as pd
+import openmc
+
+#Obtaining results from statepoint file:
+with openmc.StatePoint('statepoint.10.h5') as sp:
+    fl = sp.get_tally(name="Flux spectrum")
```

Comment:
It's certainly possible that this filename may vary by use case.  Is it also possible that the tally name might vary?

Perhaps these should be command line variables using `argparse`, with default values as shown here.